src/i18n/es.json

```diff
@@ -28,6 +28,7 @@
   },
   "forgot_password": {
     "back_login": "¿Recordaste tu contraseña y quieres regresar?",
+    "confirmation_message_HTML": "<p>¡Revisa tu casilla!</p><p>Encontrarás un Email con los pasos a Seguir.</p>",
```

Comment:
`seguir` instead of `Seguir`. 
Why not use one `p` tag?

Reply:
Because there are two paragraph.

Reply:
> seguir instead of Seguir.

I am fixing it in #171